internal/api/promote_to_stage_subscribers_v1alpha1_test.go

```diff
@@ -15,21 +15,23 @@ import (
 	svcv1alpha1 "github.com/akuity/kargo/pkg/api/service/v1alpha1"
 )
 
-func TestPromoteSubscribers(t *testing.T) {
+func TestPromoteToStageSubscribers(t *testing.T) {
 	testCases := []struct {
 		name       string
-		req        *svcv1alpha1.PromoteSubscribersRequest
+		req        *svcv1alpha1.PromoteToStageSubscribersRequest
 		server     *server
 		assertions func(
-			*connect.Response[svcv1alpha1.PromoteSubscribersResponse],
+			*testing.T,
```

Comment:
At this rate we will be done changing all of them in no time 😄

Reply:
Trying to do them every time I touch a test for other reasons. 😄